pyproject.toml

```diff
@@ -19,7 +19,14 @@ dependencies = ["spacy~=3.0", "clinlp>=0.6"]
 
 [project.optional-dependencies]
 test = ["pytest", "pytest-cov"]
-dev = ["black", "flake8", "isort", "rsconnect-python"]
+dev = ["black", "flake8", "flake8-pyproject", "isort", "rsconnect-python"]
```

Comment:
You can probably also remove rsconnect, I just included it in the template, since we almost always use it